kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/MergeQueries.java

```diff
@@ -481,6 +479,10 @@ static String batchClearQuery(TableId intermediateTable, int batchNumber) {
         .toString();
   }
 
+  public void waitRandomTime(int sleepMs, int jitterMs) throws InterruptedException {
```

Comment:
SleepUtils looks like the right place for this. Any reasons for moving it to MergeQueries?

Reply:
That was because the current sleep call in MergeQueries had hardcoded values and static so it was annoying for testing, since BigqueryWriter has a method waitRandomTime I made the same thing in MergeQueries so I was able to modify the sleep call in test.
But now we use config for sleepMs and maxRetry so the problem is not here anymore I recreated SleepUtils